Libraries/Pressability/Pressability.js

```diff
@@ -491,8 +491,9 @@ export default class Pressability {
       },
 
       onResponderMove: (event: PressEvent): void => {
-        if (this._config.onPressMove != null) {
-          this._config.onPressMove(event);
+        const {onPressMove} = this._config;
+        if (onPressMove != null) {
+          onPressMove(event);
         }
```

Comment:
```suggestion
        !!onPressMove && onPressMove(event)
```
We could probably use this logic as well I'm pretty sure

Reply:
Yeah, but I was trying to mimic the pattern used in the rest of file, for example:
* https://github.com/facebook/react-native/blob/master/Libraries/Pressability/Pressability.js#L717
* https://github.com/facebook/react-native/blob/master/Libraries/Pressability/Pressability.js#L440
* https://github.com/facebook/react-native/blob/master/Libraries/Pressability/Pressability.js#L589

But I can rewrite/replace all the current usages to the line you have proposed in the separate commit, please @kevkevinpal let me know, what's the decision 🙂

Reply:
I'm not sure if it's that way for readability sake since my proposed way is a bit harder to understand if you don't get the logic maybe someone else can answer that but otherwise the pr looks good ☺️